### PR TITLE
[serverless] Attack Discovery UI updates

### PR DESCRIPTION
**Pull request by benironside**
Part of #5889 — updates screenshots for Attack Discovery as well as references to the number of alerts it can process and the default alerts that get sent to the LLM.

Preview: Go to the link in [this comment](https://github.com/elastic/security-docs/pull/5956#issuecomment-2420308823) then go to Security -> View serverless docs -> AI for security -> Attack Discovery

### CODE REVIEW

**Comment by elasticdocs** [comment]
:rocket: Built _elastic-dot-co-docs-preview-docs_ successfully!

* https://elastic-dot-co-docs-production-doatg2otc-elastic-dev.vercel.app

* Built with commit eff45f13024dfb1502c71b2a5bd12bce74602761

> Issues? Visit #next-docs in Slack

File: docs/serverless/AI-for-security/attack-discovery.mdx [hunk 1]
```diff
@@ -34,17 +34,17 @@ When you access Attack discovery for the first time, you'll need to select an LL
 2. Select an existing connector from the dropdown menu, or add a new one. 
 
 <DocCallOut title="Recommended models">
-While Attack discovery is compatible with many different models, our testing found increased performance with Claude 3 Sonnet and Claude 3 Opus. In general, models with larger context windows are more effective for Attack discovery.
+While Attack discovery is compatible with many different models, our testing found increased performance with Claude 3 Sonnet. In general, models with larger context windows are more effective for Attack discovery.
```
Comment (andrew-goldstein) [inline docs/serverless/AI-for-security/attack-discovery.mdx:37]: Consider `Claude 3.5 Sonnet`

File: docs/AI-for-security/attack-discovery.asciidoc [hunk 2]
```diff
@@ -48,7 +48,7 @@ When you access Attack discovery for the first time, you'll need to select an LL
 While Attack discovery is compatible with many different models, our testing found increased performance with Claude 3 Sonnet and Claude 3 Opus. In general, models with larger context windows are more effective for Attack discovery.
```
Comment (benironside) [inline docs/AI-for-security/attack-discovery.asciidoc:48]:
```suggestion
While Attack discovery is compatible with many different models, our testing found increased performance with Claude 3.5 Sonnet. In general, models with larger context windows are more effective for Attack discovery.
```

File: docs/serverless/AI-for-security/attack-discovery.mdx (hunk 1 above)
Comment (benironside) [inline docs/serverless/AI-for-security/attack-discovery.mdx:37]:
```suggestion
While Attack discovery is compatible with many different models, our testing found increased performance with Claude 3.5 Sonnet. In general, models with larger context windows are more effective for Attack discovery.
```

File: docs/serverless/AI-for-security/attack-discovery.mdx [hunk 3]
```diff
@@ -34,17 +34,17 @@ When you access Attack discovery for the first time, you'll need to select an LL
...
+![Attack discovery empty state](../images/attack-discovery/attck-disc-select-model-empty-state.png) 
 
 3. Once you've selected a connector, click **Generate** to start the analysis. 
 
 It may take from a few seconds up to several minutes to generate discoveries, depending on the number of alerts and the model you selected. 
 
 <DocCallOut title="Important">
-Attack discovery is in technical preview and will only analyze opened and acknowleged alerts from the past 24 hours. By default it only analyzes up to 20 alerts within this timeframe, but you can expand this up to 100 by going to **AI Assistant → Settings (<DocIcon type="gear" title="settings icon"/>) → Knowledge Base** and updating the **Alerts** setting.
+Attack discovery is in technical preview and will only analyze opened and acknowleged alerts from the past 24 hours. By default it analyzes up to 100 alerts within this timeframe, but you can expand this up to 500 by clicking the settings icon <DocIcon type="gear" title="Settings icon"/> next to the model selection menu. Note that sending more alerts than your chosen LLM can handle may result in an error.
```
Comment (nastasha-solomon) [inline docs/serverless/AI-for-security/attack-discovery.mdx:47]: Tiny nit -  after users click the settings icon, do they need to take further action to increase the number of alerts that Attack discovery can analyzer?

**Review by nastasha-solomon — APPROVED** [review]
Left one question that needs your input. Everything else looks good though!

**Review by andrew-goldstein — DISMISSED** [review]
Thanks @benironside for these updates! 
LGTM 🚀

**Review by nastasha-solomon — DISMISSED** [review]

**Review by nastasha-solomon — APPROVED** [review]

**Pull request merged**